app/src/main/res/values/strings.xml

```diff
@@ -30,6 +30,8 @@
     <string name="toosmallerr">Value is too small. Minimum value: </string>
     <string name="toolargeerr">Value is too large. Maximum value: </string>
     <string name="baddatatypeerr">Answer must be numeric</string>
+    <string name="reloadsurveyslabel">Reload all surveys</string>
```

Comment:
With this revert, there is no need to change other language strings right?

Reply:
No, they did not get pulled yet